### PR TITLE
Adding architecture from Landsat_C2_SRST nw-poudre-historical branch and the Landsat_C2_SRST_template

### PR DESCRIPTION
**Pull request by steeleb**
Hey Katie, not much to review here (aside from the updated readme).

Most of the LOC here are part of code which has already been reviewed and has been integrated into the ROSSyndicate repos. The config file is from [this repo branch](https://github.com/rossyndicate/Landsat_C2_SRST/tree/nw-poudre-historical), and then the targets workflow and functions originated from the [template repo](https://github.com/rossyndicate/Landsat_C2_SRST_template). 

Minor changes related to copying file: I've renamed the _targets.R from the template repo to b_RS_data_acquisition.R and the data_acquisition/ path to b_RS_data_acquisition/. 

This code WILL NOT RUN. Progress on #8, this is meant to create a clean slate to document any actual changes to the workflow that are pertinent to this dataset.

Please only review the readme file, which has been updated with the additional targets groups.

Anika, tagging you to get you accustomed to PR stuff for when I do want you to read/provide feedback for written aspects of projects! We can talk through this more next week!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,23 +33,27 @@ This targets workflow is broken down into groups of target lists that perform
...
 for RS retrieval. The group of functions collates a few different polygon and 
 point files into a single file of each type as needed for the RS workflow.
 
 
-__b_historical_RS_data_collation__:
+_b_RS_data_acquisition_:
+
+This group of targets acquires the Landsat record for our focus lakes as part of 
+the Northern Water project, our internal Cache La Poudre lakes, as well as all
+lakes greater than 1ha in the CLP HUC12.
```
Comment (kathryn-willi) [inline README.md]: hectacre or hectare?

**Review by kathryn-willi — APPROVED** [review]
World's shortest PR! README Looks good!

File: README.md (hunk 1 above)
Comment (steeleb) [inline README.md]:
```suggestion
lakes greater than 1 hectare in the CLP HUC12.
```

**Comment by steeleb** [comment]
Thank you @kathryn-willi !

**Pull request merged**